examples/ransomware_detection/README.md

```diff
@@ -72,10 +72,12 @@ Run the following from the `examples/ransomware_detection` directory to start th
 ```bash
 python run.py --server_url=localhost:8001 \
               --sliding_window=3 \
-              --model_name=ransomw-model-short-rf \
+              --model_name=ransomware_model_tl \
```

Comment:
Could we include some high-level information about the model, including details about the dataset used for training? Additionally, it would be greatly beneficial to explain how to generate the dataset for training the models or specify the required data for running the inference pipeline.

Would be valuable to create a notebook that demonstrates how to train the model using a sample dataset and run it through the pipeline to showcase its ransomware capabilities. 

Providing an explanation of the output structure generated by the pipeline would greatly enhance the comprehensibility of the documentation. Adding this information to the documentation would be much appreciated.